stdlib/Sockets/src/addrinfo.jl

```diff
@@ -251,3 +251,38 @@ function getipaddr()
     ccall(:uv_free_interface_addresses, Cvoid, (Ptr{UInt8}, Int32), addr, count)
     return lo_present ? localhost : error("No networking interface available")
 end
+
+"""
+    getipaddrs() -> IPv4[]
```

Comment:
The convention for this in docstrings is to list the type itself rather than an instance of it, so this should be
```suggestion
    getipaddrs() -> Vector{IPv4}
```